embedded-aerospike/pom.xml

```diff
@@ -14,8 +14,8 @@
     <artifactId>embedded-aerospike</artifactId>
 
     <properties>
-        <aerospike-client.version>4.4.18</aerospike-client.version>
-        <joda-time.version>2.10.8</joda-time.version>
+        <aerospike-client.version>5.1.3</aerospike-client.version>
```

Comment:
due to internal concerns, we want to stick with the old version of aerospike, same for aerospike client, please revert this section for now